Python/Product/Uwp/Templates/Projects/BackgroundService/PythonBackgroundService.vstemplate

```diff
@@ -4,8 +4,8 @@
     <Description>A project for creating an Python IoT Background Application</Description>
     <Icon Package="{6dbd7c1e-1f1b-496d-ac7c-c55dae66c783}" ID="402" />
     <ProjectType>Python</ProjectType>
-    <SortOrder>20</SortOrder>
-    <NumberOfParentCategoriesToRollUp>2</NumberOfParentCategoriesToRollUp>
+    <SortOrder>2000</SortOrder>
```

Comment:
Still leaving this at 2000 in case we eventually do need to do rollups and any subsequent templates can be 2000 based